internal/app/scanner_run/scanner_run_events.go

```diff
@@ -0,0 +1,21 @@
+// SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Greenhouse contributors
+// SPDX-License-Identifier: Apache-2.0
+
+package scanner_run
+
+import (
+	"github.com/cloudoperators/heureka/internal/app/event"
+	"github.com/cloudoperators/heureka/internal/entity"
+)
+
+const (
+	CreateScannerRunEventName event.EventName = "CreateScannerRun"
```

Comment:
I think here we need more event types: 
- one for whenever the Scanner _started_
- one for whenever the Scanner _finished_
- (optional) one for whenever the Scanner _failed_

Reply:
You are correct. As stated in the Slack message, this is a partial PR, so I'll add the events in the future, when the implementation for the effects of the events is done.